lib/nile/src/Nile.ts

```diff
@@ -44,6 +45,7 @@ export class NileApi {
       this.workspaces.workspace = workspace;
       this.organizations.workspace = workspace;
       this.authz.workspace = workspace;
+      this.metrics.workspace = workspace;
```

Comment:
Not really related to this, but... Why do we need to set workspace on all the things? Can't we have some shared Context or something that they have access to? Seems like a strange pattern to me